hack/chains/verify-attestation-task.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: verify-attestation-signature
```

Comment:
I'm wondering if it is too soon to define this task over in https://github.com/redhat-appstudio/build-definitions/tree/main/tasks ?